rotkehlchen/chain/evm/decoding/hop/decoder.py

```diff
@@ -380,6 +380,10 @@ def _decode_common_liquidity(self, context: DecoderContext, first_token_raw: byt
             return None
 
         if (swap_asset_id := self.swaps_to_asset.get(context.tx_log.address)) is None:
+            log.error(
+                f'Could not find asset for the saddle swap address {context.tx_log.address!r} '
```

Comment:
This is not what we need to add in such logs. The important things are missing. The things we need to know to try and debug. Those are:

1. Chain name (notice this is deployed in many chains)
2. Transaction hash.


The rest is not necessary as we would see it the moment we debug and they would instead flood the logs with too munch info.